src/monitoring/bin/db_full_check.sh

```diff
@@ -0,0 +1,8 @@
+echo CROMWELL_GOOGLE_PROJECT is ${CROMWELL_GOOGLE_PROJECT}
+echo CROMWELL_CLOUDSQL_INSTANCE is ${CROMWELL_CLOUDSQL_INSTANCE}
+echo CROMWELL_MAXIMUM_DATABASE_SIZE_TIB is ${CROMWELL_MAXIMUM_DATABASE_SIZE_TIB}
+echo CROMWELL_DATABASE_FORECAST_LIMIT_MONTHS is ${CROMWELL_DATABASE_FORECAST_LIMIT_MONTHS}
+# echo CROMWELL_METRICS_ACCESS_TOKEN is ${CROMWELL_METRICS_ACCESS_TOKEN}
+pip install numpy requests python-dateutil
```

Comment:
1. shouldn't think be pip3? ... but even better:
2. could this all happen in a docker container?

Reply:
It is happening [in a Docker container](https://github.com/broadinstitute/dsp-jenkins/compare/mlc_database_fullness?expand=1). For now it's in happening from an artisanally crafted Jenkins job, I'm still waiting on help with how to `dsl-seed-job` the Groovy in that PR on that particular Jenkins.